charts/kargo/values.yaml

```diff
@@ -144,6 +144,11 @@ api:
   probes:
     ## @param api.probes.enabled Whether liveness and readiness probes should be included in the API server deployment. It is sometimes advantageous to disable these during local development.
     enabled: true
+    livenessProbe:
+      initialDelaySeconds: 10
+
```

Comment:
Nit: Killing this blank line would visually make it more obvious that `readinessProbe` below is a component of this block.